lib/plugins/aws/provider/awsProvider.js

```diff
@@ -123,6 +124,54 @@ const impl = {
   },
 };
 
+const baseAlbAuthorizerProperties = {
+  onUnauthenticatedRequest: { enum: ['allow', 'authenticate', 'deny'] },
+  requestExtraParams: {
+    type: 'object',
+    maxProperties: 10,
+    additionalProperties: { type: 'string' },
+  },
+  scope: { type: 'string' },
+  sessionCookieName: { type: 'string' },
+  sessionTimeout: { type: 'integer' },
```

Comment:
This seems not addressed https://github.com/serverless/serverless/pull/8291#discussion_r495843839